docs/3.tutorials/auction/0-intro.md

```diff
@@ -0,0 +1,117 @@
+---
+id: introduction
+title: A Step-by-Step Guide to Mastering NEAR
+sidebar_label: Introduction
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Welcome! In this guide we will help you navigate NEAR tech stack, so you can build Web3 applications from start to finish in no-time.
+
+We'll start from a simple auction contract and slowly build on top of it a full Web3 application to carry on-chain auctions.
+
+By the time you finish this tutorial, you will have learned how to use several key primitives and concepts along the way:
+
+- Building and testing a contract
+- Deploying, updating and locking a contract
+- Creating a frontend to interact with the contract
+- Using an indexing API to keep track of the contract's activity
+- Creating a factory to deploy new contracts
```

Comment:
could be nice to link these items to other articles or sections for direct reference